pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opengis.cite</groupId>
   <artifactId>ets-common</artifactId>
-  <version>10-SNAPSHOT</version>
+  <version>9</version>
```

Comment:
Version 9 exists already. The version will be updated by the next release. Please remove this modification.